lib/stripe.js

```diff
@@ -133,6 +135,26 @@ Stripe.prototype = {
     );
   },
 
+  setAppInfo: function(info) {
+    if (info && typeof info !== 'object') {
+      throw new Error('AppInfo must be an object.');
+    }
+
+    // Kill the cached UA string because it may no longer be valid
+    Stripe.USER_AGENT_SERIALIZED = undefined;
+
+    this._appInfo = APP_INFO_PROPERTIES.reduce(function(accum, prop) {
```

Comment:
@brandur-stripe Do you mean ensuring that if it contains _something_, then it must contain _at least_ `name`?